docs/api-proposal/proposed-routes.md

```diff
@@ -0,0 +1,76 @@
+This doc proposes a new set of API endpoints for the multi-jurisdiction audit flow.
+
+Some principles I tried to follow:
+
+- Reuse existing routes when available
+- Use REST conventions
+- Avoid reusing routes across roles. Instead of reusing one route and returning different scoped responses, make different roles query different endpoints with explicit scoping in the route. That way we just have to check if a user has access to the data they are asking for, as opposed to trying to filter data based on their access. (I learned this one the hard way.)
+
+New routes are in **bold**.
+
+# Audit Admin: Audit setup
+
+![](./create-audit.png)
+
+- POST /election/new - create new audit
+- **GET /election** - get all audits
+
+![](upload-jurisdiction-file.png)
+
+- POST /election/<election_id>/jurisdiction/csv - upload CSV file of jurisdictions with JAs
+- GET /election/<election_id>/jurisdiction/csv - download CSV file of jurisdictions with JAs
```

Comment:
Technically these endpoints exist already. I’d be wary of hard-coding the expected format in the URL.

Reply:
A few thoughts:
- I was thinking we could rename them now since they aren't used yet. Will be harder later.
- I was thinking of the file upload as part of the jurisdictions API, which is why I was thinking of nesting it under `/jurisdiction/`
- Fair point about hardcoding the format. Though I generally think once we make an API with a format, we're kind of stuck with it, so that's why I was thinking about being explicit. If we wanted to change it, we'd probably have to make a new endpoint anyway.

But I don't have a strong opinion here

Reply:
With all of this, it is harder to change later but since we're the only client we can change things as long as @MorganLove knows we're doing it.